### PR TITLE
[RFC][NI]visually-hidden helper

### PR DESCRIPTION
**Pull request by fredcerdeira**
New helper to hide elements while at the same time make them accessible

### CODE REVIEW

File: src/styles/_common/_helpers.scss [hunk 1]
```diff
@@ -31,3 +31,13 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
```
Comment (AndreJoaquim) [inline src/styles/_common/_helpers.scss]: newline

Comment (fredcerdeira) [inline src/styles/_common/_helpers.scss]: ?

Comment (fredcerdeira) [inline src/styles/_common/_helpers.scss]: Some files does not have empty lines to separate the different mixins.

Comment (AndreJoaquim) [inline src/styles/_common/_helpers.scss]: I think it's a good standard to have, nonetheless

Comment (fredcerdeira) [inline src/styles/_common/_helpers.scss]: Agree, though is it needed to make these changes on all of them.

Comment (fredcerdeira) [inline src/styles/_common/_helpers.scss]: Done in this file.

**Pull request merged**